dolphinscheduler-ui/src/js/conf/home/pages/projects/pages/instance/pages/list/_source/list.vue

```diff
@@ -422,6 +448,18 @@
           })
         }
       },
+      /**
+       * Resume from forced success
+       * @param RESUME_FROM_FORCED_SUCCESS
+       */
+      _resumeFromForcedSuccess (item, index) {
+        this._countDownFn({
+          id: item.id,
+          executeType: 'RESUME_FROM_FORCED_SUCCESS',
```

Comment:
There is no execution type called `RESUME_FROM_FORCED_SUCCESS` in `dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/ExecuteType.java`, so it seems that submitting this execution type here will cause some problem.